web/src/components/feature/package_dialog/ComponentTable.tsx

```diff
@@ -36,23 +54,14 @@ export const ComponentTable = ({
             value={componentRatios[component.id] ?? ''}
             pattern="^\d+(\.\d+)?([eE][-+]?\d+)?$"
             variant={
-              ratiosAreValid[component.id] === false ? 'warning' : undefined
+              ratiosAreValid[component.id] === false ||
+              (component.id === '5' && !(Number(componentRatios['5']) > 0))
```

Comment:
But I would rather have this special case logic inside "handleChange".
It's inside that function we set if ratios are valid or not.

They way it is written now. You can not trust "ratiosAreValid" and need to do extra checks several places

Reply:
Perhaps renaming ratiosAreValid to inputsAreValidNumbers then. The solution you propose is more complicated, since it means we would need to add extra logic to the onOptionsChange for the autocomplete in ComponentSelector.